test/inductor/test_compiled_optimizers.py

```diff
@@ -82,66 +88,55 @@ class KernelCounts(NamedTuple):
 }
 
 
-def build_compiled_opt_kwarg_db():
+def build_opt_kwarg_db():
     compiled_opt_db = []
     for optim_info in optim_db:
         if optim_info.optim_cls not in KERNEL_COUNTS:
             continue
 
         for device in ["cpu", "cuda"]:
-            for optim_inputs in optim_info.optim_inputs_func(device):
-                for foreach in [True, False]:
-                    kwargs = dict(optim_inputs.kwargs)
-                    name = (
-                        f"test_{optim_info.optim_cls.__name__.lower()}"
-                        f"{'_foreach' if foreach else ''}"
+            for optim_inputs in _get_optim_inputs_including_global_cliquey_kwargs(
+                device, None, optim_info, skip=("differentiable", "fused")
```

Comment:
I see you are now skipping this--we do need this for full coverage eventually, so will fused coverage come in another PR?

Reply:
It's covered in the parity tests .. ?

Reply:
Oh..so do we no longer care that fused leads to 0 kernels generated? I thought that was the key reason to have included it earlier.

Reply:
nah I don't care anymore, if it works it's good haha. If it compiles one kernel I don't mind either